doc/en/components/grids/pivot-grid/overview.md

```diff
@@ -2,13 +2,368 @@
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/{PivotGridSample}-overview"
+           alt="{Platform} Pivot Grid with Pivot Selector Example">
+</code-view>
+
+## Getting started with {Platform} Pivot Grid
+
+The {Platform} {PivotGridName} can be configured via the `PivotConfiguration` property.
+
+```html
+<igx-pivot-grid #grid1 [data]="data" [pivotConfiguration]="pivotConfigHierarchy">
+</igx-pivot-grid>
+```
+
+```razor
+<IgbPivotGrid PivotConfiguration="PivotConfiguration" Data="PivotData">
+</IgbPivotGrid>
+```
+
+It is defined by three main dimensions: **rows**, **columns** and **values**. The **rows** and **columns** define the grouped structure that is displayed in the rows and columns of the grid. The **values** define the aggregation fields and the aggregation that will be used to calculate and display the related values of the groups.
+
+A filter can also be defined via the **filters** configuration property. It can be used for fields that you do not want to add as a dimension or a value but would like to filter their related member values via the UI.
+
+### Dimensions configuration
+
+Each basic dimension configuration requires a `MemberName` that matches a field from the provided **data**.
+<!-- Angular -->
+Or a `MemberFunction` that extracts a value from the record in case of complex objects or other custom scenarios.
+<!-- Angular -->
+
+Multiple sibling dimensions can be defined, which creates a more complex nested group in the related row or column dimension area.
+
+The dimensions can be reordered or moved from one area to another via their corresponding chips using drag & drop.
+
+A dimension can also describe an expandable hierarchy via the `ChildLevel` property, for example:
+```typescript
+   {
+            memberFunction: () => 'All',
+            memberName: 'AllProducts',
+            enabled: true,
+            childLevel: {
+                memberFunction: (data) => data.ProductCategory,
+                memberName: 'ProductCategory',
+                enabled: true
+            }
+    }
+```
+
+```razor
+@code {
+    var pivotConfiguration = new IgbPivotConfiguration();
+    pivotConfiguration.Rows.Add(new IgbPivotDimension()
+        {
+            MemberName = "Product",
+            Enabled = true,
+            Name = "pivotDimension1",
+            ChildLevel = new IgbPivotDimension() { MemberName = "Country", Enabled = true, Name = "pivotDimension2" }
+        });
+}
+```
+In this case the dimension renders an expander in the related section of the grid (row or column) and allows the children to be expanded or collapsed as part of the hierarchy. By default the row dimensions are initially expanded. This behavior can be controlled with the `DefaultExpandState` property of the pivot grid.
+
+### Predefined dimensions
+
+As part of the pivot grid some additional predefined dimensions are exposed for easier configuration:
```

Comment:
There are multiple places where `Pivot Grid` is written in small case `pivot grid`